scripts/quickstart.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
why change this?

Reply:
That's exactly the point why I started with `sh`, but I think both are OK.

Reply:
I'm asking Taha for his opinion

Reply:
I think `/bin/bash` is a safer option: in most distros, `/bin/sh` is a symlink to `/bin/bash`. In some distros, `/bin/sh` behaves exactly like `/bin/bash` while in others it operates in a compatibility mode that disables some advanced bash features (even though it actually bash running the script). Because of that, I find that the behavior of `/bin/sh` is unpredictable: some things might work in one distro but not work in another.